ptsprojects/zephyr/gap_wid.py

```diff
@@ -216,7 +216,14 @@ def hdl_wid_51(desc):
 
 
 def hdl_wid_52(desc):
-    hdl_wid_51(desc)
+    stack = get_stack()
+
+    btp.gap_adv_off()
```

Comment:
Is it necessary to disable advertising here?

Reply:
yes, similar to hdl_wid_90. The new configuration is taken by restarting advertisement.